src/helperFunctions/web_interface.py

```diff
@@ -77,3 +78,10 @@ def __exit__(self, *args):
 def get_template_as_string(view_name):
     path = os.path.join(get_template_dir(), view_name)
     return get_binary_from_file(path).decode('utf-8')
+
+
+def is_superuser(user):
+    if isinstance(user._get_current_object(), AnonymousUser):  # auth disabled
```

Comment:
it would be more intuitive if this functionality was put in its own function (e.g. "auth_is_enabled(user)"), since it has little to do with whether or not the user is a "superuser"
it would then need to be tested first (e.g. "if not auth_is_enabled(user) or is_superuser(user)")

Reply:
user_has_admin_clearance()